### ISSUE TITLE
zephir_parse_file  terminates the program without any error messages

### ISSUE THREAD

**Issue by sergeyklay** [phalcon/php-zephir-parser#19]
_From @pakey on March 21, 2017 16:46_

There is no error feedback after executing `zephir build`
After checking found in the implementation to the `Library/CompilerFile.php` file
```
$ir = zephir_parse_file(file_get_contents($zepRealPath), $zepRealPath);
```
The program is terminated, no one after any output, and can not capture the specific error message,
What should I do if I do not know where I can do it?

_Copied from original issue: phalcon/zephir#1409_

**Comment by sergeyklay** [comment]
Try to check log file.

**Comment by sergeyklay** [comment]
_From @pakey on March 22, 2017 0:54_

the log file is empty

**Comment by sergeyklay** [comment]
Is it posible to see source to reproduce?

**Comment by sergeyklay** [comment]
Fixed in the [**`v1.0.3`**](https://github.com/phalcon/php-zephir-parser/releases/tag/v1.0.3).
